cmd/bucket-handlers.go

```diff
@@ -383,6 +383,11 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 		return
 	}
 
+	if s3Error := checkRequestAuthType(ctx, r, policy.DeleteObjectAction, bucket, ""); s3Error != ErrNone {
+		writeErrorResponseHeadersOnly(w, errorCodes.ToAPIErr(s3Error))
+		return
+	}
+
 	// Before proceeding validate if bucket exists.
```

Comment:
This is incorrect check @wthorp - We need this behavior to comply with S3 API behavior.